models/articles.models.js

```diff
@@ -1,11 +1,24 @@
-const db = require('../db/connection')
+const db = require("../db/connection");
 
 exports.selectArticleById = (articleId) => {
-    return db.query(`SELECT * FROM articles WHERE article_id = $1`, [articleId])
-    .then(({rows}) => {
-        if(rows.length === 0) {
-            return Promise.reject({status: 404, msg: 'Not found'})
-        }
-        return (rows[0])
-    })
-}
\ No newline at end of file
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1`, [articleId])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Not found" });
```

Comment:
Worth specifying that it's an article that is not found here instead of leaving it generic - you know what exactly will trigger this error so you can be explicit